packages/wallet-frontend/src/pages/account/[accountId]/[paymentPointerId].tsx

```diff
@@ -103,67 +97,35 @@ export const getServerSideProps: GetServerSideProps<{
     }
   }
 
-  const [accountResponse, paymentPointerResponse, transactionsResponse] =
-    await Promise.all([
-      accountService.get(result.data.accountId, ctx.req.headers.cookie),
-      paymentPointerService.get(
-        result.data.accountId,
-        result.data.paymentPointerId,
-        ctx.req.headers.cookie
-      ),
-      paymentPointerService.getTransactions(
-        result.data.accountId,
-        result.data.paymentPointerId,
-        ctx.req.headers.cookie
-      )
-    ])
+  const [accountResponse, paymentPointerResponse] = await Promise.all([
```

Comment:
adding payment pointers as part of account response it is useful in some cases(you make one request instead of 2), but some times the response will have data that you don't need
for example here payment pointers are not needed in account response and if you have a lot of payment pointers that can be a significant increase in response time

Reply:
on transactions we have order parameter in the request and we will need to add pagination and some other filters, so not sure how much sense it has to change the list of transactions request and have them included in payment pointer response instead

Reply:
Makes a lot of sense. Will keep things as is for the moment.